kore/src/Kore/Internal/Substitution.hs

```diff
@@ -493,6 +494,11 @@ isSimplified _ (Substitution _) = False
 isSimplified sideCondition (NormalizedSubstitution normalized) =
     all (TermLike.isSimplified sideCondition) normalized
 
+isSimplifiedIgnoreCondition :: Substitution variable -> Bool
```

Comment:
Please add documentation similar to `Kore.Attribute.Pattern.Simplified.isSimplifiedIgnoreCondition`.